virtcontainers/vm.go

```diff
@@ -475,7 +475,7 @@ func (v *VM) ToGrpc(config VMConfig) (*pb.GrpcVM, error) {
 
 func (v *VM) GetVMStatus() *pb.GrpcVMStatus {
 	return &pb.GrpcVMStatus{
-		Pid:    int64(v.hypervisor.pid()),
+		Pid:    int64(v.hypervisor.pid()[0]),
```

Comment:
needs to check if len more than 0

Reply:
We need to document that the array ensure the  pid of hypervisor is on index 0.
If we are planing to break the API I wonder if we could do something additional. 

- virtiofsd is not depedent of qemu we potencially use in other hypervisors.
- we may have or not additional process that associated with a sandbox that work together with the VMM.

- We have the option to add `AdditionalPids() -> Map String:int`  to the hypervisor  interface. 
And then iterate over that map without need to break the current semantic of `pid()`
 OR make `pid()` return a map just in case we want the caller need an specific field (like this case).

I prefer `AdditionalPids()`

Reply:
@jcvenegas I'm following @stefanha's suggestion to rename it getPids(). The purpose is quite similar so let's not split it.